automation/publish.sh

```diff
@@ -6,9 +6,40 @@
 # IMAGE_REPO
 # To run it just do proper docker login and automation/publish.sh
 
+image_registry=${IMAGE_REGISTRY:-quay.io}
+image_repo=${IMAGE_REPO:-nmstate}
+branch=${PULL_BASE_REF:-$(git rev-parse --abbrev-ref HEAD)}
+docs_source_branch=master
+docs_container=${image_registry}/${image_repo}/kubernetes-nmstate-docs:$docs_source_branch
+
+# Publish kubernets-nmstate containers
 source automation/check-patch.setup.sh
-cd ${TMP_PROJECT_PATH}
-make \
-    IMAGE_REGISTRY=${IMAGE_REGISTRY:-quay.io}  \
-    IMAGE_REPO=${IMAGE_REPO:-nmstate} \
-    push-handler
+(
+    cd ${TMP_PROJECT_PATH}
+    make \
+        IMAGE_REGISTRY=${image_registry}  \
+        IMAGE_REPO=${image_repo} \
+        push-handler
+)
+
+if [ "$branch" != $docs_source_branch ]; then
```

Comment:
Could you change it to a positive check? The issue with this is that you make everything after this line "documentation script. If I add something to the bottom, it would be skipped everywhere except on the docs source branch.